nifi-registry/nifi-registry-core/nifi-registry-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -630,6 +646,40 @@ In this example, the users and groups are loaded from LDAP but the servers are m
 from an attribute in a LDAP entry based on the `User Identity Attribute`. The `NiFi Identity` values are established in the local file using the
 `Initial User Identity` properties.
 
+===== DatabaseAccessPolicyProvider
+Like `DatabaseUserGroupProvider` described above, this type of PolicyProvider takes the DB connection details from `nifi-registry.properties` file. 
```

Comment:
Same as above, recommend spelling out "database" rather than "DB", for consistency and clarity.